### ISSUE TITLE
Prevent adding the same authors as pairs

### ISSUE THREAD

**Issue by augustohp** [augustohp/git-pair#4]
We should prevent people from adding the same author as a pair or two pair authors with the same name:

Expected behaviour is as follows:

``` sh
$ git pair Nelson
Local authors name now are: Augusto Pascutti + Nelson
$ git pair Nelson
You are already pairing with "Nelson"
$ git pair reset
Local author name now is: Augusto Pascutti
$ git pair "Augusto Pascutti"
You cannot pair with yourself, can you?
```

**Comment by augustohp** [comment]
I am not closing this issue since the error message is specific to only the original author, we still miss an specific message to author being pairing with. @malukenho, will you accept the "challenge"?

**Comment by malukenho** [comment]
@augustohp yeah! I'll try this.

**Comment by augustohp** [comment]
Closed on 9a4c98c633f81f3f47fea3d28c370454210fe04a.
